packages/calypso-e2e/src/lib/pages/people-page.ts

```diff
@@ -125,7 +123,7 @@ export class PeoplePage {
 	async selectInvitedUser( emailAddress: string ): Promise< void > {
 		await Promise.all( [
 			this.page.waitForNavigation(),
-			this.page.click( selectors.invitedUser( emailAddress ) ),
+			this.page.click( selectors.invitedUser( emailAddress ), { force: true } ),
```

Comment:
I believe this argument bypasses actionability checks - was there a specific reason for doing that? That seems like something that might cause other false-positives, I would think we might want to use the auto-waiting here!

Reply:
My rationale for force clicking is that, despite waiting for network requests to complete, the vanilla `click` method is sometimes not able to locate the invited user.

It appears to me that the page is not fully done rendering all invited users even when the network requests are over, and if there are a large number of invited users this discrepancy grows larger.

With that said, I am still a bit unsure why the vanilla `click` is not able to locate the selector for a recently invited user as they would likely be at the top of the invite list.

Reply:
We can try removing the force click part and observe how the invites behave. 

We can also remove this step from the spec as well, as it isn't super important that we check the invites we _just sent_ show up immediately.